src/qml/views/filter/AttachedFilters.qml

```diff
@@ -255,7 +255,12 @@ Rectangle {
                 anchors.fill: attachedFiltersView.contentItem
                 z: 1
                 onClicked: mouse => {
-                    filterClicked(attachedFiltersView.indexAt(mouseX, mouseY));
+                   let index = attachedFiltersView.indexAt(mouseX, mouseY)
+                   if(index < 0) {
```

Comment:
Space missing between if and (

Reply:
here is the bug video, and I added space  between if and （

Reply:
> Space missing between if and (

I replied to your email with the bug video